testing/fpgainfo/test_fpgainfo_c.cpp

```diff
@@ -1015,13 +1017,14 @@ TEST_P(fpgainfo_c_p, get_metrics1) {
     fpga_metric_info metrics_info[64];
     fpga_metric metrics[64];
     uint64_t num_metrics;
+    uint64_t num_metrics_info;
     uint32_t matches = 0;
 
     ASSERT_EQ(fpgaGetProperties(NULL, &filter), FPGA_OK);
     ASSERT_EQ(fpgaPropertiesSetObjectType(filter,FPGA_DEVICE), FPGA_OK);
     ASSERT_EQ(fpgaEnumerate(&filter, 1, &token, 1, &matches), FPGA_OK);
 
-    EXPECT_EQ(get_metrics(token, POWER, metrics_info, metrics, &num_metrics), FPGA_OK);
+    EXPECT_EQ(get_metrics(token, POWER, metrics_info, &num_metrics_info, metrics, &num_metrics), FPGA_OK);
```

Comment:
POWER is too generic a name.  We should add an appropriate prefix.

Reply:
fixed